src/test/java/elements/Search.java

```diff
@@ -0,0 +1,25 @@
+package elements;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Search {
+
+    WebDriver driver;
+    String label;
+    String locator = "//*[contains(text(),'%s')]/ancestor::*[@class='slds-form-element__control']//input";
+    String selectOption = "//*[contains(text(),'%s')]/ancestor::div[contains(@role,'listbox')]";
```

Comment:
better to replace "div" to * (asterisk) if possible

Reply:
fix